File: My-Blog-ServeOpen/appone/models/article.py
from django.contrib.auth.models import User
from django.db import models

from appone.models.label import Label
from appone.models.sort import Sort

class Article(models.Model):
    user_id = models.IntegerField(verbose_name='用户ID')
    sort_id = models.IntegerField(verbose_name='分类ID')
    label_id = models.IntegerField(verbose_name='标签ID')
    article_cover = models.URLField(max_length=256, null=True, blank=True, verbose_name='封面')
    article_title = models.CharField(max_length=32, verbose_name='博文标题')
    article_content = models.TextField(verbose_name='博文内容')
    view_count = models.IntegerField(default=0, verbose_name='浏览量')
    like_count = models.IntegerField(default=0, verbose_name='点赞数')
    view_status = models.BooleanField(default=True, verbose_name='是否可见')
    password = models.CharField(max_length=128, null=True, blank=True, verbose_name='密码')
    recommend_status = models.BooleanField(default=False, verbose_name='是否推荐')
    comment_status = models.BooleanField(default=True, verbose_name='是否启用评论')
    create_time = models.DateTimeField(auto_now_add=True, verbose_name='创建时间')
    update_time = models.DateTimeField(auto_now=True, verbose_name='最终修改时间')
    update_by = models.CharField(max_length=32, null=True, blank=True, verbose_name='最终修改人')
    deleted = models.BooleanField(default=False, verbose_name='是否删除')

    class Meta:
        db_table = 'article'
        verbose_name = '文章表'
        verbose_name_plural = verbose_name

    def __str__(self):
        return str(self.id) + '-' + str(self.article_title)

File: My-Blog-Go/models/article.go
package models

import (
	"time"
)

// Article 文章表
type Article struct {
	ID              uint      `gorm:"primaryKey;column:id" json:"id"`
	UserID          uint      `gorm:"column:user_id" json:"userId"`
	SortID          uint      `gorm:"column:sort_id" json:"sortId"`
	LabelID         uint      `gorm:"column:label_id" json:"labelId"`
	ArticleCover    *string   `gorm:"column:article_cover;size:256" json:"articleCover,omitempty"`
	ArticleTitle    string    `gorm:"column:article_title;size:32" json:"articleTitle"`
	ArticleContent  string    `gorm:"column:article_content;type:text" json:"articleContent"`
	Summary         string    `gorm:"column:summary;type:text" json:"summary"`
	ViewCount       int       `gorm:"column:view_count;default:0" json:"viewCount"`
	LikeCount       int       `gorm:"column:like_count;default:0" json:"likeCount"`
	ViewStatus      bool      `gorm:"column:view_status;default:true" json:"viewStatus"`
	Password        *string   `gorm:"column:password;size:128" json:"password,omitempty"`
	RecommendStatus bool      `gorm:"column:recommend_status;default:false" json:"recommendStatus"`
	CommentStatus   bool      `gorm:"column:comment_status;default:true" json:"commentStatus"`
	CreateTime      time.Time `gorm:"column:create_time;autoCreateTime" json:"createTime"`
	UpdateTime      time.Time `gorm:"column:update_time;autoUpdateTime" json:"updateTime"`
	UpdateBy        *string   `gorm:"column:update_by;size:32" json:"updateBy,omitempty"`
	Deleted         bool      `gorm:"column:deleted;default:false" json:"deleted"`
}

func (Article) TableName() string {
	return "article"
}

// ArticleLike 文章点赞表
type ArticleLike struct {
	ID        uint `gorm:"primaryKey;column:id" json:"id"`
	UserID    uint `gorm:"column:user_id_id" json:"userId"`
	ArticleID uint `gorm:"column:article_id_id" json:"articleId"`
}

func (ArticleLike) TableName() string {
	return "like"
}
